packages/winpanda/extra/src/winpanda/svcm/nssm.py

```diff
@@ -25,6 +25,8 @@
 
 LOG = logger.get_logger(__name__)
 
+CMD_KILL = "taskkill /f /fi \"SERVICES eq {}\""
```

Comment:
If the service exits after the timeout but before we run this kill command, will this raise an error? It looks like maybe no, since the command uses a filter, but, for example, does `taskkill /f /fi "SERVICES eq does-not-exist"` exit successfully?

Reply:
In a case of KILL debug message will be added.
Instead of curly brakets service name that can not be stpped successfuly will be placed and we will not have `does-not-exist` case.
When we are talking about filter this command will came from stop where we already have the filter and also we are not using `_primitive_command` method where we do have this filter. As a option I can do this method as a static and will sent service name there then it will be like a function with predefined powershell commad and I will use it in `stop` method.